content/docs/tutorials/handling-errors.mdx

```diff
@@ -0,0 +1,241 @@
+> Try, try again.  
+
+The old adage rings true in this case. There are many possible scenarios (for example: `504 Timeout`s, transient outages, congestion on the Stellar network) in which retrying your transaction submission is the only viable fallback. This should be considered the last line of defense, though; often-times, the error can be rectified by a [safe modification](#request-adjustments) to the transaction.
+
+Since there's no mechanism to "cancel" a transaction after it has been submitted, the first key to successful retries is leveraging [timebounds](https://developers.stellar.org/docs/glossary/transactions/#time-bounds): though optional, timebounds allow you to introduce a bit of determinism into an innately non-deterministic system. If the timebound has been exceeded, the transaction has a definitive, final state: either it made it into a ledger or it timed out. The second key comes from a certain Horizon guarantee: **you can safely retry submission as long as you don't modify the transaction**. Specifically (excerpt is from [this example](https://developers.stellar.org/docs/tutorials/send-and-receive-payments/#send-a-payment)),
+
+> If the transaction has already been successfully applied to the ledger, Horizon will simply return the saved result and not attempt to submit the transaction again. Only in cases where a transaction’s status is unknown (and thus will have a chance of being included into a ledger) will a resubmission to the network occur.  
+
+In other words, if your transaction makes it through at any point in time, you'll get your previously-submitted transaction back for all subsequent retries.
+
+
+#### Example Scenario
+Suppose you submit a transaction and it enters the queue of the Stellar network, but Horizon crashes while giving you a response. Uncertain about the transaction status, you resubmit (with no changes!) until either (a) Horizon comes back up to give you a reply or (b) the timebounds you set get exceeded. There are only two possible results: the transaction makes it into a ledger (exactly once!) and Horizon gives you the response, or the transaction never makes it out of the queue and you receive the corresponding `tx_too_late` response.
+
+#### Example Implementation
+
+<CodeExample>
+
+```js
+let server = sdk.Server("horizon.stellar.org");
+
+function submitTransaction(tx, timeout) {
+  // If a maximum timebound isn't set, we should definitely set one.
+  const maxTime = tx.maxTime || 20;
+  tx.maxTime = maxTime;
+
+  return server.submitTransaction(tx).catch(function (error) {
+    if (isNonRetryErrorCase(error)) {
+      // ...do other error handling...
+      return;
+    }
+    
+    timeout = timeout || 1; // start the back-off process
+    return sleep(timeout).then(function () {
+      return submitTransaction(tx, timeout + 5);
+    });
+  });
+}
+```
+
+(We assume the existence of a `sleep` implementation akin to the one [here](https://stackoverflow.com/a/39914235).)
+
+</CodeExample>
```

Comment:
Can't have text inside a `CodeExample`, I think this is probably what's causing the error
```suggestion
</CodeExample>

(We assume the existence of a `sleep` implementation akin to the one [here](https://stackoverflow.com/a/39914235).)
```

Reply:
Dunno how I overlooked this.. thank you!